.deploy/monitoring/rules.yml

```diff
@@ -1,20 +1,27 @@
-# Default alerting rules.
-# - Vault collateralization is below the secure threshold.
-# - A redeem has less than an hour until it expires.
+# Default Kintsugi/Interlay vault alerting rules.
 groups:
 - name: VaultMonitoring
   rules:
   - alert: BelowSecureThreshold
-    expr: collateralization < 2.6
-    for: 1s
+    expr: collateralization < 2.6 and collateralization > 0.01
+    for: 1m
     annotations:
-      description: 'Vault with currency pair {{ $labels.currency }} is below the Secure Threshold.'
+      description: 'Vault {{ $labels.job }} with currency pair {{ $labels.currency }} and {{ $value }} collateralization is below the Secure Threshold.'
```

Comment:
1) if the metrics logic could be extended to provide more values for Prometheus rules to trigger against, that'd be cool (nothing specific to suggest at the moment. Having just learned Prometheus in April, the rules capabilities still seem primitive)
2) I run multiple vaults for each currency pair, with each job named something like `vault1_CD`, so that's why the currency pair alone is insufficient for my setup. So when I print the job name it has been the best way to communicate exactly which service was having an issue.